integration_tests/Makefile

```diff
@@ -1,8 +1,11 @@
-test-setup: egg-info bus-test bus-documentation
+test-setup: egg-info bus-test bus-documentation asyncapi-cli-pull
 
 test:
 	pytest -x
 
+asyncapi-cli-pull:
+	docker pull asyncapi/cli
```

Comment:
If you want the docker-compose file to rely only local images pulled from the Makefile, you could use a specific name when pulling. e.g.
```bash
docker pull asyncapi/cli:latest
docker tag asyncapi/cli:latest local/asyncapi/cli
```
then use `local/asyncapi/cli` in the compose file.

Reply:
In the past, we removed all pulled logic from Makefile. wazo-test-helper already take care of it
For dev you can use `docker compose pull`

Reply:
You mean `wazo-test-helper` pulls images itself based on the docker-compose files?

Reply:
I'll rewrite to use the `pull_containers` method instead and remove the one in the makefile.  Will be more in line with other wazo services

Reply:
Yep seems so. if we want want docker-compose to work without a make invocation then ignore my comment.